server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/SetMessagesCreationProcessor.java

```diff
@@ -271,9 +285,13 @@ private Flags getMessageFlags(CreationMessage message) {
         return result;
     }
 
-    private void sendMessage(MailboxMessage mailboxMessage, Message jmapMessage, MailboxSession session) throws MessagingException, IOException {
-        Mail mail = mailFactory.build(mailboxMessage, jmapMessage);
-        MailMetadata metadata = new MailMetadata(jmapMessage.getId(), session.getUser().getUserName());
-        mailSpool.send(mail, metadata);
+    private void sendMessage(MailboxMessage mailboxMessage, Message jmapMessage, MailboxSession session) throws MessagingException {
+        try {
+            Mail mail = mailFactory.build(mailboxMessage, jmapMessage);
+            MailMetadata metadata = new MailMetadata(jmapMessage.getId(), session.getUser().getUserName());
+            mailSpool.send(mail, metadata);
+        } catch (IOException e) {
+            Throwables.propagate(e);
```

Comment:
I'm not sure this is caught later. Maybe this can be transformed into a MailboxException? or a MessagingException?